quiche/src/recovery/rtt.rs

```diff
@@ -33,6 +33,8 @@ pub(crate) const INITIAL_RTT: Duration = Duration::from_millis(333);
 
 pub(crate) const RTT_WINDOW: Duration = Duration::from_secs(300);
 
+const JUMP_THRESHOLD: u32 = 3;
```

Comment:
Why do we set this to 3? Would it make sense to make this constant configurable?